src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -53,6 +53,13 @@ export const CheckImg = styled.img`
     width: 147px;
   }
 
+  @media (min-width: 1023px) {
+    margin-left: auto;
+    position: relative;
+    left: 5vw;
+    width: 100px;
+  }
+
   @media (min-width: 1200px) {
     margin-left: auto;
     position: relative;
```

Comment:
is not declare position relative again since you've declared position relative before, in the line 50